src/types/track.ts

```diff
@@ -18,6 +18,12 @@ export type NFTTrackJoin = {
   processedTrackId: string;
 }
 
+export type Artworks = {
+  id: number;
+  cid: string;
```

Comment:
cid is optional (eg: if there's an error), also error is optional. but at least one of the two is required.

so to be super pedantic, could have a Union type of cidOrError = {cid:string } |  {error:string }

and smth like
```
Artwork = {
  trackId: string;
  size: strin;
} & cidOrError
```